spec/app_spec.rb

```diff
@@ -200,4 +200,27 @@
     expect(response['current_page']).to eq 2
   end
 
+  it "should return a list of addresses that have been updated since a certain date" do
+    Timecop.freeze(DateTime.new(2013,1,1))
+
+    5.times do
+      FactoryGirl.create(:address)
+    end
+
+    Timecop.return
+
+    date = DateTime.now
+
+    5.times do
+      FactoryGirl.create(:address)
+    end
+
+    get 'addresses', { updated_since: date }
```

Comment:
Good catch. It does auto transform it into an ISO8601 string, but I've added a `to_s` just to be more descriptive